saleor/graphql/core/types/common.py

```diff
@@ -178,6 +178,11 @@ class OrderError(Error):
         description="List of product variants that are associated with the error",
         required=False,
     )
+    order_lines = graphene.List(
```

Comment:
In this type, we have `order_line` and `order_lines`. IMO we should have one of these.

Reply:
I dropped `orderLine` and left only `orderLines`